index.bs

```diff
@@ -580,22 +580,22 @@ are [=content script config=]s.
 The {{HTMLControlledFrameElement/partition}} attribute takes an identifier
 specifying where data related to the Controlled Frame's instance should be
 stored. The identifier is composed of a string of alphanumeric characters.
-All data for the [=embedded navigable=] will be stored in a [=storage shelf=]
+All data for the [=embedded navigable=] are stored in a [=storage shelf=]
```

Comment:
I think Reilly's intention here was that we use "must be" to match https://datatracker.ietf.org/doc/html/rfc2119